pkgs/tf.nix

```
@@ -0,0 +1,21 @@
+{stdenv, source, fetchurl}:  
```

```suggestion
{stdenv, source}:  
```
`fetchurl` is not used